source/http/HttpConnection.cpp

```diff
@@ -290,6 +280,8 @@ namespace Aws
             {
             }
 
+            HttpClientStream::~HttpClientStream() {}
```

Comment:
c++ trivia: you can also write `HttpClientStream::~HttpClientStream() = default;` in the cpp file.